app/controllers/templates_controller.rb

```diff
@@ -63,7 +64,7 @@ def destroy
   private
 
   def template_params
-    params.require(:template).permit(:name, :question_ids => [])
+    params.require(:template).permit(:name, :user_id, :question_ids => [])
   end
```

Comment:
we don't need ```user_id``` here.
in fact this param will introduce security issue - it will be possible to create tempates for other users by changing value of this param.
use
current_user.tempates.build or something like that